seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-jdbc/src/main/java/org/apache/seatunnel/flink/source/JdbcSource.java

```diff
@@ -116,41 +128,86 @@ public void prepare(FlinkEnvironment env) {
         if (config.hasPath(SOURCE_FETCH_SIZE)) {
             fetchSize = config.getInt(SOURCE_FETCH_SIZE);
         }
+        if (config.hasPath(PARALLELISM)) {
+            parallelism = config.getInt(PARALLELISM);
+        } else {
+            parallelism = env.getBatchEnvironment().getParallelism();
+        }
+        try {
+            Class.forName(driverName);
+            Connection connection = DriverManager.getConnection(dbUrl, username, password);
```

Comment:
The connection has not been closed here, it's better to close the connection in the finally or use try(Connection connection = xx)